config/config.go

```diff
@@ -32,6 +32,8 @@ var (
 	}
 
 	defaultExecutionTime = Duration(30 * time.Second)
+
+	defaultMaxPayloadSize = ByteSize(100000000)
```

Comment:
don't forget in the doc to talk about the default max value

Reply:
nb: given what @Garnek20 saw in production (for the chatamart cluster), this default size is still to small. IHMO, if we want to have a transparent release (meaning a user that was not using this feature don't understand why some queries are not cached anymore), we should put a very high and unrealistic value like 1<<50.

Reply:
@sigua-cs Can you add a comment explaining why this value is set so high?